examples/config/grid_config.config

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0"?>
+
+<plugin filename="MinimalScene">
+    <ignition-gui>
+      <title>View 1</title>
+      <property type="string" key="state">docked</property>
+    </ignition-gui>
+    <engine>ogre2</engine>
+    <scene>scene</scene>
+    <ambient_light>1 1 1</ambient_light>
+    <background_color>0.8 0.8 0.8</background_color>
+    <camera_pose>-6 3 6 0 0.5 0</camera_pose>
+</plugin>
+<plugin filename="InteractiveViewControl" name="Interactive view control">
+  <ignition-gui>
+    <anchors target="View 1">
+      <line own="right" target="right"/>
+      <line own="top" target="top"/>
+    </anchors>
+    <property key="resizable" type="bool">false</property>
+    <property key="width" type="double">5</property>
+    <property key="height" type="double">5</property>
+    <property key="state" type="string">floating</property>
+    <property key="showTitleBar" type="bool">false</property>
+  </ignition-gui>
+</plugin>
+<plugin filename="GridConfig" name="Grid config">
+  <ignition-gui>
+    <property type="string" key="state">docked</property>
+  </ignition-gui>
+  <cell_count>3</cell_count>
```

Comment:
How about renaming to `<horizontal_cell_count>` for clarity?

Reply:
I added that as an alias, so that both `cell_count` and `horizontal_cell_count` are accepted, with `horizontal_cell_count` overriding in case both are set.

The reasons to keep `cell_count` are:

* That corresponds to the `ign-rendering` API call
* That's backwards compatible with `Grid3D`